en/devtools-usage.md

```diff
@@ -162,19 +162,41 @@ When a project is generated using developer tools. A configuration file can be f
 
 Change the database section in your config.ini file:
 
-```ini
-[database]
-adapter  = Mysql
-host     = "127.0.0.1"
-username = "root"
-password = "secret"
-dbname   = "store_db"
-
-[phalcon]
-controllersDir = "../app/controllers/"
-modelsDir      = "../app/models/"
-viewsDir       = "../app/views/"
-baseUri        = "/store/"
+```php
+<?php
+/*
+ * Modified: prepend directory path of current file, because of this file own different ENV under between Apache and command line.
+ * NOTE: please remove this comment.
```

Comment:
I am not sure I understand the meaning of these comments. Can you give me a little more detail.

Reply:
ping @NickCoolii

Reply:
@ninjapanzer @sergeyklay 
this comment was in generated by Phalcon config.

Reply:
@NickCoolii Just remove this comment. We'll try to sort out a bit later. Sorry it took this long